extensions/eclipse-che-theia-plugin-ext/src/node/che-client-ip-service.ts

```diff
@@ -18,9 +18,8 @@ export class CheClientIpServiceContribution implements BackendApplicationContrib
     configure(app: express.Application): void {
         app.get('/che/client-ip', (req, res) => {
             res.json({
-                ip: req.connection.remoteAddress,
-                ipFamily: req.connection.remoteFamily,
-                port: req.connection.remotePort
+                ip: req.get('x-forwarded-for') !== undefined ? req.get('x-forwarded-for') : req.connection.remoteAddress,
+                port: req.get('x-forwarded-port') !== undefined ? req.get('x-forwarded-port') : req.connection.remoteAddress,
```

Comment:
`: req.connection.remoteAddress` is it correct?

Reply:
thanks